--- launcher/server.pro
+++ launcher/server.pro
@@ -2,7 +2,7 @@ TEMPLATE	= app
 
 CONFIG		+= qt warn_on 
 
-DESTDIR		= $$(OPIEDIR)/bin
+DESTDIR		=
 
 HEADERS		+= server.h \
 		  qrr.h \
@@ -20,7 +20,7 @@ HEADERS		+= server.h \
 		  shutdownimpl.h \
 		  launcher.h \
 		  launcherview.h \
-		  $$(OPIEDIR)/core/apps/calibrate/calibrate.h \
+		  calibrate/calibrate.h \
 		  startmenu.h \
 		  transferserver.h \
 		  qcopbridge.h \
@@ -31,7 +31,7 @@ HEADERS		+= server.h \
 		  serverapp.h \
 		  qprocess.h \
 		  screensaver.h \
-		  $$(OPIEDIR)/noncore/settings/mediummount/mediumwidget.h \
+		  mediummount/mediumwidget.h \
                   mediadlg.h
 
 SOURCES		+= server.cpp \
@@ -50,7 +50,7 @@ SOURCES		+= server.cpp \
 		  shutdownimpl.cpp \
 		  launcher.cpp \
 		  launcherview.cpp \
-		  $$(OPIEDIR)/core/apps/calibrate/calibrate.cpp \
+		  calibrate/calibrate.cpp \
 		  transferserver.cpp \
 		  packageslave.cpp \
    		  irserver.cpp \
@@ -63,18 +63,18 @@ SOURCES		+= server.cpp \
 		  qprocess.cpp \
 		  qprocess_unix.cpp \
 		  screensaver.cpp \
-		  $$(OPIEDIR)/noncore/settings/mediummount/mediumwidget.cc \
+		  mediummount/mediumwidget.cc \
                   mediadlg.cpp
 
 
-INCLUDEPATH += $(OPIEDIR)/core/apps/calibrate
-DEPENDPATH	+= $(OPIEDIR)/core/apps/calibrate
+INCLUDEPATH += calibrate
+DEPENDPATH	+= calibrate
 
-INCLUDEPATH += $(OPIEDIR)/include $(OPIEDIR)/rsync
-DEPENDPATH	+= $(OPIEDIR)/rsync
+INCLUDEPATH += $(OPIEDIR)/include rsync
+DEPENDPATH	+= rsync
 
-INCLUDEPATH += $(OPIEDIR)/noncore/settings/mediummount
-DEPENDPATH  += $(OPIEDIR)/noncore/settings/mediummount
+INCLUDEPATH += mediummount
+DEPENDPATH  += mediummount
 
 LIBS        += -lqpe -lopiecore2 -lopieui2 -lopiesecurity2 -lqrsync -lsysfs -lcrypt -lm
 TARGET       = qpe
@@ -84,6 +84,7 @@ TARGET       = qpe
 
 include( $(OPIEDIR)/include.pro )
 
+CONFIG += LAUNCHER_SYNC_V2
 contains( CONFIG, LAUNCHER_SYNC_V2 ){
     DEFINES += OPIE_SYNC_V2
     SOURCES += virtualfs.cpp \
